### PR TITLE
Documents how to get started with SQLite

### LINKED ISSUE #511: Document how to get started with SQLite
docker-compose is noted in our readme as the easiest way to get started.

However, docker-compose is pinned to cassandra, as is our other [quick-start](https://github.com/openzipkin/zipkin/blob/master/doc/mac-quickstart.md).

@irff and I'm sure others would like to use SQLite. It should have a README file saying how to setup this on your laptop, possibly also using docker-compose. The [existing db doc](https://github.com/openzipkin/zipkin/blob/master/doc/sql-databases.md) should be updated and referenced.

### PR DESCRIPTION
**Pull request by codefromthecrypt**
This cleans up scripts that use SBT to run local zipkin processes. Most
notably, it updates the README with full instructions up to generating
fake trace data.

Closes #511

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
FYI @irff @dsyer @spencergibb

File: README.md [hunk 1]
```diff
@@ -29,13 +29,22 @@ $ docker-compose up
 ```
 
 If you are happy building from source you can use the scripts in the
-[`bin/`] directory of this repository. E.g. to start a collector
-pointing to an existing local redis server:
-
-```
-$ git clone https://github.com/openzipkin/zipkin
-$ cd zipkin
-$ ./bin/collector-redis
```
Comment (codefromthecrypt) [inline README.md:38]: I took this out, as the simplest start is to use SQLite (as it doesn't need an external process running). Ack that until tonight, that didn't work :)

File: bin/collector-redis [hunk 2]
```diff
@@ -1,2 +0,0 @@
-#!/usr/bin/env sh
-bin/sbt 'project zipkin-collector-service' 'run -f zipkin-collector-service/config/collector-redis.scala'
```
Comment (codefromthecrypt) [inline bin/collector-redis:2]: `bin/collector-redis` has the same character count as `./bin/collector redis` :p

File: bin/git-pull-request.rb [hunk 3]
```diff
@@ -1,315 +0,0 @@
-#!/usr/bin/env ruby
```
Comment (codefromthecrypt) [inline bin/git-pull-request.rb:1]: this isn't something we should host

**Comment by spencergibb** [comment]
:+1:

File: README.md [hunk 4]
```diff
@@ -29,13 +29,22 @@ $ docker-compose up
...
+# start the collector server in a new terminal session or tab
+$ ./bin/collector
+# start the query server in a new terminal session or tab
+$ ./bin/query
+# start the web server in a new terminal session or tab
+$ ./bin/web
+# create dummy traces
+$ ./bin/tracegen
+# open the ui and look at them!
+$ open http://localhost:8080/
```
Comment (abesto) [inline README.md:47]: <3

**Comment by abesto** [comment]
:shipit:

**Comment by codefromthecrypt** [comment]
thnx for the review @abesto and @spencergibb!

**Pull request merged**